test/toAdditive.lean

```diff
@@ -96,7 +96,7 @@ if some_def.in_namespace then x * x else x
 
 -- cannot apply `@[to_additive]` to `some_def` if `some_def.in_namespace` doesn't have the attribute
 run_cmd do
-  Lean.Elab.Command.liftCoreM <| successIfFail (ToAdditive.transformDecl `Test.some_def `Test.add_some_def)
+  Lean.Elab.Command.liftCoreM <| fail_if_success (ToAdditive.transformDecl `Test.some_def `Test.add_some_def)
```

Comment:
This change doesn't make any sense, `fail_if_success` is a tactic not a term. In any case it doesn't belong in this docs-only PR.

Reply:
Oops, thanks. Mixed up my commits. :-(